### ISSUE TITLE
Going to the hierarchy view between 2 dialogs asking to add repeats

### ISSUE THREAD

**Issue by dbemke** [getodk/collect#6284]
#### ODK Collect version
the store version 2024.2.0, the master version a9c57b94bd7d935e4a62cfd9052066eb15ec7328 (also 2024.2.x )
#### Android version
10, 14
#### Device used
Redmi 9T, Pixel 7a
#### Problem description
It’s possible to go to the hierarchy view between 2 dialogs asking to add repeats. After going to the hierarchy view there’s a group name and "nothing to display” info. If a user tries to delete a group after steps described below the app crashes.
![repeat](https://github.com/user-attachments/assets/1287bfd0-6dd1-42e3-93a0-cd63b424368d)

#### Steps to reproduce the problem
1. Open the form.
[basicNestedRepeat(1).xlsx.txt](https://github.com/user-attachments/files/16349164/basicNestedRepeat.1.xlsx.txt)
2. Fill the first and the second question.
3. Swipe to the next question and add a repeat.
4. Fill the question.
5. Swipe to the next question.
6. In the dialog tap "do not add”.
7. After that very quickly tap the arrow to go to the hierarchy view (so that you see the view in the screenshot above).
8. Tap the bin icon and delete the group.

**Comment by seadowg** [comment]
@dbemke I'm guessing by "very quickly" you mean almost instantly right? You have to beat the animation here I think.

**Comment by dbemke** [comment]
> "very quickly" you mean almost instantly right

Yes. just tapping one thing after another. I noticed it while testing so it was rather my average speed of tapping.

**Comment by seadowg** [comment]
@grzesiek2010 I'm thinking for this (and probably a few other bugs), we should block the options menu somehow while "loading" during form entry. What do you think?
